vowpalwabbit/lda_core.cc

```diff
@@ -85,7 +85,7 @@ struct lda
 // if it makes it into VS 2015 change the next ifdef to check Visual Studio Release
 
   // static constexpr float underflow_threshold = 1.0e-10f;
-  inline const float  underflow_threshold() { return 1.0e-10f; }
+  inline float  underflow_threshold() { return 1.0e-10f; }
```

Comment:
What's going on with const here?

Reply:
const is semantically meaningless when returning POD values as they get copied.